eval/eval.go

```diff
@@ -412,7 +448,12 @@ func (s *State) evalExpressions(exps []ast.Node) ([]object.Object, *object.Error
 }
 
 func (s *State) evalIdentifier(node *ast.Identifier) object.Object {
+	// local var can shadow extensions.
 	val, ok := s.env.Get(node.Literal())
+	if ok {
+		return val
+	}
+	val, ok = s.extensions[node.Literal()]
 	if !ok {
 		return object.Error{Value: "<identifier not found: " + node.Literal() + ">"}
```

Comment:
Not related to the current PR, but an sprintf would be easier to read than concatenation

Reply:
well not for 2 strings imo (removed the `+ ">"` now so should be better)